docs/api/README.md

```diff
@@ -15,3 +15,6 @@ The APIs overview tables is here:
 |       ^        | ^                                                                                                                                                 | [LinearSearch](../api/search.md#linearsearch-rpc)<br>[LinearSearchByID](../api/search.md#linearsearchbyid-rpc)<br>[StreamLinearSearch](../api/search.md#streamlinearsearch-rpc)<br>[StreamLinearSearchByID](../api/search.md#streamlinearsearchbyid-rpc)<br>[MultiLinearSearch](../api/search.md#multilinearsearch-rpc)<br>[MultiLinearSearchByID](../api/search.md#multilinearsearchbyid-rpc) | ^                                    |
 | Remove Service | Remove the exists vector(s) from the Vald Agent Pods.                                                                                             | [Remove](../api/remove.md#remove-rpc)<br>[StreamRemove](../api/remove.md#streamremove-rpc)<br>[MultiRemove](../api/remove.md#multiremove-rpc)                                                                                                                                                                                                                                                  | [Vald Remove APIs](../api/remove.md) |
 | Object Service | Get information of vector(s) indexed into the Vald Agent Pods.                                                                                    | [Exists](../api/object.md#exists-rpc)<br>[GetObject](../api/object.md#getobject-rpc)<br>[StreamGetObject](../api/object.md#streamgetobject-rpc)                                                                                                                                                                                                                                                | [Vald Object APIs](../api/object.md) |
+
+And, Vald provides the public client libraries for some programming languages.
+If there are no client library for your favorite programming language, you can build yourself according to [build proto docs](../api/build_proto.md).
```

Comment:
```suggestion
Vald provides the public client libraries for some programming languages.
If we do not provide the client library for your favorite programming language, please refer to [build proto docs](../api/build_proto.md) to build it on your environment.
```

Reply:
I think using `we do not provide` is a sense of discomfort :thinking: